src/app/members/components/MemberList.tsx

```diff
@@ -97,7 +99,9 @@ export const MemberList = () => {
           });
         }
       }
-    });
+    };
+
+    fetchData();
```

Comment:
can we use `IIFE` instead of this method of calling function

Reply:
Thanks @Mubashar1009 Its better approach